packages/ui/.storybook/components/SelectedColorPalette.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { ColorPalette, ColorItem } from '@storybook/addon-docs/blocks';
+import { colors } from '../../src/theme/overwrites/foundations/colors';
```

Comment:
🤔 It would be great to use the theme instead of using colors directly. This way we wouldn't block the possibility of implementing dark mode in the future.